prometheus-exporters/openstack-exporter/alerts/manila.alerts

```diff
@@ -0,0 +1,22 @@
+groups:
+- name: manila
+  rules:
+    # 1. substract reserved capacity from reported overall free capacity to get usable capacity
+    # 2. only look at hardware state "live"
+    # 3. sum by host, i.e. look at NetApp cluster, summing up the aggregates aka pools
+    # 4. look at availability zones, joined in trhough manila service metrics
+  - alert: ManilaAvailabilityZoneUsageHigh
+    expr: count by(zone) (sum by (zone, host)((manila_free_capacity_gb{hardware_state="live"} - (manila_reserved_percentage/100 * manila_total_capacity_gb)) * on(host) group_left(zone) manila_service_state{service="manila-share"}) > 5*1024) < 3 # less than 3 clusters per zone with more than 5 TB free capacity
```

Comment:
metrics `(manila_free_capacity_gb{hardware_state="live"} - (manila_reserved_percentage/100 * manila_total_capacity_gb)) * on(host) group_left(zone) manila_service_state{service="manila-share"}` does not seem to work. the labels `host` and `zone` are not matched

Reply:
moved conversation to slack to illustrate on some concrete examples

Reply:
metrics deployed only to bronze, and it does work in `na-us-3`